src/components/Header/Header.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  FC,
+  useState,
+  ChangeEvent,
+  FormEvent,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onAdd: (title: string) => void,
+  tempTodo: Todo | null,
+  activeTodos: number,
+  handleUpdatingAll: () => void,
+};
+
+export const Header: FC<Props> = (props) => {
+  const {
+    onAdd,
+    tempTodo,
+    activeTodos,
+    handleUpdatingAll,
+  } = props;
+
+  const [title, setTitle] = useState('');
+
+  const handleSubmit = async (event: FormEvent) => {
+    event.preventDefault();
+
+    // added await, but ts says that it has no effect on the type of this expression
```

Comment:
Sure 'cause you didn't specified that this function is async (returns Promise<void>) in props type